frontend/src/Views/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,37 @@
+import Button from '@Commons/Button/Button.tsx';
+import {
+  StyledContent,
+  StyledButtonGroup,
+  StyledDivider,
+  StyledLandingContainer,
+} from './LandingPage.styles.ts';
+
+const LandingPage = () => (
+  <StyledLandingContainer>
+    <StyledContent>
+      <h2>About the nutrient calculator</h2>
+      <p>
+        For BC berry farmers, who maintain soil nutrient levels, Better Berries is a simple nutrient
+        calculator that lets you manage your crops&apos; on the go. Our application is intuitive to
+        use and mobile friendly, allowing you to manage your soil nutrients effectively.
+      </p>
+    </StyledContent>
+    <StyledButtonGroup>
+      <Button
+        size="lg"
+        text="New Calculation"
+        disabled={false}
+        path="/main"
+      />
```

Comment:
React router has a component for URL's. I wouldn't use a button for this. 
Buttons should be in-page logic, URLs for going somewhere different. 

Screen readers hate this one trick

Reply:
Created a ticket to fix this